src/udev/cdrom_id/cdrom_id.c

```diff
@@ -1064,16 +1064,18 @@ int main(int argc, char *argv[]) {
                 goto exit;
         }
 
-        for (int cnt = 20; cnt > 0; cnt--) {
-                struct timespec duration;
+        for (int cnt = 0; cnt < 20; cnt++) {
+                if (cnt != 0) {
+                        struct timespec duration;
+
+                        duration.tv_sec = 0;
+                        duration.tv_nsec = (100 * 1000 * 1000) + (random_u32() % 100 * 1000 * 1000);
+                        nanosleep(&duration, NULL);
```

Comment:
This is a lot more readable I think:
```
(void) usleep(100 * USEC_PER_MSEC + random_u64() % (100 * USEC_PER_MSEC));
```

(and the scheduler isn't nanosecond accurate anyway, hence the µs granularity vs ns granularity shouldn't matter.